.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*]
```

Comment:
Add `max_line_length` (only available in some editors but good for clarity).

Is suggest 120 since that's what most lines adhere to now.

Reply:
I suggest not to limit line length and use automatic wrapping dependent on window size present in any editor.